SimGeneral/MixingModule/python/digitizersCosmics_cfi.py

```diff
@@ -34,30 +35,38 @@
   )
 )
 
-theDigitizersValid = cms.PSet(
-  pixel = cms.PSet(
-    pixelDigitizer
-  ),
-  strip = cms.PSet(
-    stripDigitizer
-  ),
-  ecal = cms.PSet(
-    ecalDigitizer
-  ),
-  hcal = cms.PSet(
-    hcalDigitizer
-  ),
-  castor  = cms.PSet(
-    castorDigitizer
-  ),
-  mergedtruth = cms.PSet(
-    trackingParticles
-  )
-)
-
 from Configuration.Eras.Modifier_run3_common_cff import run3_common
 run3_common.toModify( theDigitizers, castor = None )
-run3_common.toModify( theDigitizersValid, castor = None )
```

Comment:
this probably slipped from a rebase, as undoes part of https://github.com/cms-sw/cmssw/pull/26510/commits/ef1aa4d42a427dc3537c2b965d78dbff358c6013